samples/IdentityServer.ClientSample/Startup.cs

```diff
@@ -66,11 +67,8 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
         {
             app.UseHttpsRedirection();
 
-            if (env.IsDevelopment())
-            {
-                app.UseDeveloperExceptionPage();
-            }
-            
+            if (env.IsDevelopment()) app.UseDeveloperExceptionPage();
```

Comment:
I usually don't like these on one line, what do you think?

Reply:
If it’s less than 120 chars wide I am fine with a one liner, but I still like to use braces.

Reply:
Hi, thanks this one is fixed now.

Reply:
My bad, I missed the brace preference. I usually prefer single line statements/expressions to be without braces but we can put in preference for.

Reply:
I have changed the editorconfig to have ```csharp_prefer_braces = true:suggestion```